src/features/generic_events/aggregate/index.js

```diff
@@ -90,12 +106,12 @@ export class Aggregate extends AggregateBase {
     }
 
     const eventAttributes = {
-      /** Agent-level custom attributes */
-      ...(getInfo(this.agentIdentifier).jsAttributes || {}),
       /** Common attributes shared on all generic events */
       referrerUrl: this.referrerUrl,
       currentUrl: cleanURL('' + location),
       pageUrl: cleanURL(getRuntime(this.agentIdentifier).origin),
```

Comment:
These need to go after `...obj` if they aren't meant to be changed.

Also, `timestamp` should belong to all generic events too instead of just page action.

Reply:
timestamp does belong to all generic events... you just need to scroll down slightly

Reply:
ill just put it in the merge object thats probably easier to follow